pkg/systemd/services/services.jsx

```diff
@@ -947,8 +976,8 @@ const ServicesPageFilters = ({
                              className="services-text-filter"
                              placeholder={_("Filter by name or description")}
                              value={currentTextFilter}
-                             onChange={setCurrentTextFilter}
-                             onClear={() => setCurrentTextFilter('')} />
+                             onChange={onTextFilterChanged}
+                             onClear={() => onTextFilterChanged('')} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18115-20230104-115006-dfe9a654-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
That's a good point -- we never test clearing the filter?

Reply:
I will add a test as a followup.